### ISSUE TITLE
Doesn't support escaped code points?

### ISSUE THREAD

**Issue by alin04** [gorilla/css#14]
The ["consume an escaped code point"](https://www.w3.org/TR/css-syntax-3/#consume-an-escaped-code-point0) rule doesn't appear to be implemented?

**Comment by kisielk** [comment]
I don’t believe it does. If you’re interested in adding support a PR with tests would be welcome.

**Comment by alin04** [comment]
I see #13 is almost a total rewrite that does support escapes. What's the status of the PR?

**Comment by kisielk** [comment]
Honestly I haven't had the time to review it in full or come up with a strategy on migrating the package to a new version.

**Comment by alin04** [comment]
Ok. I might add a fix to the current impl then...
